client/imports/coursepage/coursepage.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import '../topbar.css';
+ 
+@Component({
+  selector: 'coursepage',
+  templateUrl: '/client/imports/coursepage/coursepage.html'
+})
+
+export class CoursePage { 
+  title = "My courses";
+  items = fetchCourses("a");
+
+}
+
+/* fetchCourses: takes in useid and fetch the list of courses from database
+ */
+function fetchCourses(user){
+
+    // TODO : replace courses with actual tasks fetched from database
```

Comment:
1) Sander has had these data structures created for almost two weeks- this should not be a //TODO at this point.
2) Even so, your havent actually done any styling- it is just a bulleted list.